DEFINITION_OF_DONE.md

```diff
@@ -0,0 +1,20 @@
+# Definition of Done (DoD)
+
+This document is a collaborative effort among team members to establish clear criteria for completing tasks. The DoD serves as a guide to ensure the quality and completeness of work within the team.
+
+To meet our standards, every task should adhere to the following criteria:
+
+- Code has been reviewed by at least one other team member.
+- Functionality works as expected and acceptance criteria are met
+- Coding standards are followed.
+- No critical issues or bugs are present.
+- Unit tests pass successfully and the code successfully builds.
+- Code meets accessibility standards.
+- Code is well-documented.
+- README is updated with relevant information.
+- API documentation is updated.
+- Code meets performance standards.
+- Code has been checked for common security issues.
+- Sensitive information is handled securely.
+- Deployment scripts are up-to-date.
+- Deployment process is documented.
```

Comment:
What if everything is automated?

Reply:
Maybe we could merge this sentence with the one before into something like : `Deployment scripts are up-to-date and documented`. The intention here being that not only "the code" is documented but also the process and scripts that support the delivery of the lib.